docroot/themes/custom/uids_base/scss/admin.scss

```diff
@@ -66,17 +73,22 @@
 }
 
 .layout-builder__add-block {
-  outline: 2px dotted #0036b1;
-  outline-offset: -2px;
-  background-color: #f3f4f9;
-  padding: 1.5em 0;
-  margin-top: 1.5rem;
+  padding: 0;
+  margin-top: 0;
   text-align: center;
 }
 
 .layout-builder__section {
-  margin-bottom: 1.5em;
-  outline: 2px dotted #e6ecf8;
+  margin-bottom: 0;
+
+  &:hover {
+    outline: 2px dotted #0036b1;
+    outline-offset: 8px;
+  }
+
+  .hero {
+    height: 30vh;
```

Comment:
I'm not sure if this is the best way to address part of #1908.

Reply:
I think we want to do something like `height: auto;` for now until the Hero refactor is complete.  The `vh` has added specifically for the home page video to stay within the viewport.

Reply:
Yes that works much better. 👍